includes/glossary.md

```diff
@@ -39,3 +39,6 @@
 *[dataset definition]: specifies the details of data — for example, patients — that you want to include in your study. Used by Data Builder.
 *[ehrQL]: electronic health record query language. The language used by the Data Builder tool to extract cohorts
 *[OpenSAFELY Contracts]: provide a specification to make clear the data provided by different data backends, and to provide guidelines and tooling for data providers wishing to standardise their data for researchers
+*[backend]: one of several clinical databases that researchers might query via OpenSAFELY.
```

Comment:
add "data provider" - an operator of a database (in UK data protection law, a _data processor_) that contains sensitive patient data; who also provides a secure infrastructure for running opensafely next to that data

Then "backend" is "an individual clinical database provided by a data provider" (I think)

Reply:
In the "Contracts domain modelling", we used "backend implementer", which I take to mean the same as "data provider".

Any opinions here on the preferable term?

Reply:
Tough one. I'm not really keen on "backend" but for reasons I can't quite articulate. I'd say either canvas more widely (if you have appetite) or make a call yourself!

Reply:
(maybe because it makes me think of bottoms)

Reply:
I'll go back to using "data provider" which was what I originally had down. I changed this because I'd spotted while reading more about Contracts.